setup.py

```diff
@@ -57,9 +56,6 @@
     'github': [
         'github3.py>=1.0.0a4',
     ],
-    'orcid': [
```

Comment:
please keep the `'orcid'` section even when it's empty for compatibility reasons.